pkg/api/update.go

```diff
@@ -16,7 +16,7 @@
 
 package api
 
-//nolint
+// nolint
```

Comment:
Uhmm... this is a machine-readable comment-directive, not a human-readable comment. I think it should literally be `//nolint`.